apps/api/src/app/environments/environment.ts

```diff
@@ -1,333 +1,341 @@
+        },
+        syslumennService: {
+          url: process.env.SYSLUMENN_HOST,
+          username: process.env.SYSLUMENN_USERNAME,
+          password: process.env.SYSLUMENN_PASSWORD,
+        },
+        rskDomain: {
+          username: process.env.RSK_API_USERNAME,
+          password: process.env.RSK_API_PASSWORD,
+          url: process.env.RSK_API_URL,
+        },
+        icelandicNamesRegistry: {
+          backendUrl: process.env.ICELANDIC_NAMES_REGISTRY_BACKEND_URL,
+        },
+        regulationsDomain: {
+          url: process.env.REGULATIONS_API_URL,
+        },
+        endorsementSystem: {
+          baseApiUrl: process.env.ENDORSEMENT_SYSTEM_BASE_API_URL,
+        },
+        nationalRegistryXRoad: {
+          url: process.env.XROAD_BASE_PATH_WITH_ENV,
+          memberCode: process.env.XROAD_TJODSKRA_MEMBER_CODE,
+          apiPath: process.env.XROAD_TJODSKRA_API_PATH,
+          clientId: process.env.XROAD_CLIENT_ID,
+        },
+        paymentDomain: {
+          xRoadBaseUrl: process.env.XROAD_BASE_PATH,
+          xRoadProviderId: process.env.XROAD_PAYMENT_PROVIDER_ID,
+          xRoadClientId: process.env.XROAD_CLIENT_ID,
+          username: process.env.XROAD_PAYMENT_USER,
+          password: process.env.XROAD_PAYMENT_PASSWORD,
+          callbackBaseUrl: process.env.XROAD_PAYMENT_BASE_CALLBACK_URL,
+          callbackAdditionUrl: process.env.XROAD_PAYMENT_ADDITION_CALLBACK_URL,
+          arkBaseUrl: process.env.ARK_BASE_URL,
+        },
+        temporaryVoterRegistry: {
+          baseApiUrl: process.env.TEMPORARY_VOTER_REGISTRY_BASE_API_URL,
+        },
+        partyLetterRegistry: {
+          baseApiUrl: process.env.PARTY_LETTER_REGISTRY_BASE_API_URL,
+        },
+        fjarmalDomain: {
+          xroadApiPath: process.env.XROAD_FINANCES_PATH,
+          ttl: parseInt(process.env.FJARMAL_TTL, 10) || 600,
+        },
+        pkpass: {
+          apiKey: process.env.PKPASS_API_KEY,
+          apiUrl: process.env.PKPASS_API_URL,
+          secretKey: process.env.PKPASS_SECRET_KEY,
+          cacheKey: process.env.PKPASS_CACHE_KEY,
+          cacheTokenExpiryDelta: process.env.PKPASS_CACHE_TOKEN_EXPIRY_DELTA,
+          authRetries: process.env.PKPASS_AUTH_RETRIES,
+        },
+        audit: {
+          defaultNamespace: '@island.is/api',
+          groupName: process.env.AUDIT_GROUP_NAME,
+          serviceName: 'api',
+        },
+        paymentSchedule: {
+          xRoadBaseUrl: process.env.XROAD_BASE_PATH,
+          xRoadProviderId: process.env.XROAD_PAYMENT_PROVIDER_ID,
+          xRoadClientId: process.env.XROAD_CLIENT_ID,
+          username: process.env.PAYMENT_SCHEDULE_USER,
+          password: process.env.PAYMENT_SCHEDULE_PASSWORD,
+        },
+        islykill: {
+          cert: process.env.ISLYKILL_CERT,
+          passphrase: process.env.ISLYKILL_SERVICE_PASSPHRASE,
+          basePath: process.env.ISLYKILL_SERVICE_BASEPATH,
+        },
+      }
+    : {
```

Comment:
Having this huge ternary makes it hard to find where the dev config vs. prod config begins/ends.

My proposal would be to rather keep the same format as before with two variables devConfig and prodConfig and return the previous ternary at the bottom ; makes it easier to read.

Reply:
👍🏻